lib/msprime.c

```diff
@@ -449,6 +450,7 @@ msp_set_simulation_model_dirac(msp_t *self, double psi)
         goto out;
     }
     self->model.params.dirac_coalescent.psi = psi;
+    self->model.params.dirac_coalescent.const_c = 1.0; // hard coded const_c value
```

Comment:
What's this for? If it's not a parameter it shouldn't be in the model params struct. If it is a parameter we should just call it `c` and input it the same way as the others.